tests/network/conftest.py

```diff
@@ -221,6 +222,11 @@ def network_operator(admin_client):
     )
 
 
+@pytest.fixture(scope="session")
+def mtv_namespace_scope_session(admin_client: DynamicClient) -> Namespace:
```

Comment:
You can drop `scope_session` from the name, we do not need to reflect the scope.